.github/workflows/ci.yml

```
@@ -0,0 +1,72 @@
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+      - uses: pnpm/action-setup@v3
+        with:
+          version: latest
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+      - name: Run tests
+        run: pnpm test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)